app/components/primer/open_project/border_box/collapsible_header.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Primer
+  module OpenProject
+    module BorderBox
+      # Add a general description of component here
+      # Add additional usage considerations or best practices that may aid the user to use the component correctly.
+      # @accessibility Add any accessibility considerations
+      class CollapsibleHeader < Primer::Component
+        status :open_project
+
+        # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+        def initialize(title:, count: nil, description: nil, collapsed: false, box:, **system_arguments)
+          @title = title
+          @count = count
+          @description = description
+          @collapsed = collapsed
+          @box = box
+          @system_arguments = system_arguments
+
+          @system_arguments[:tag] = "collapsible-header"
+          @system_arguments[:classes] = class_names(
+            system_arguments[:classes],
+            "CollapsibleHeader",
+            "CollapsibleHeader--collapsed" => collapsed?
```

Comment:
```suggestion
            "CollapsibleHeader--collapsed" => @collapsed
```

That should be fine. The method was only added for testing purposes